<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstLaunch</class>
 <widget class="QDialog" name="FirstLaunch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Premier lancement de DadaBooks</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/programme/images/icone.png</normaloff>:/programme/images/icone.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Je veux une collection en</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonXML">
       <property name="text">
        <string>XML</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSQLite">
       <property name="text">
        <string>SQLite</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSQL">
       <property name="text">
        <string>MySQL</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Je possède déjà une collection que je souhaite importer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSkip">
       <property name="text">
        <string>Passer cette étape</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Premier lancement de DadaBooks</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nous avons détecté qu'il s'agit de la première fois que vous lancez DadaBooks.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>DadaBooks est simple et pratique à utiliser, il vous aidera à conserver vos collections ordonnées et vous permettra de trouver un livre ou un film en un seul clic.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Pour gérer votre collection, plusieurs formats sont proposés:</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>1)XML : Pratique pour une petite collection</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>2)SQLite : Pratique pour une grande collection, dispose de foncionnalités avancées comme la gestion d'auteurs/acteurs de manière séparée.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>3)MySQL/MariaDB : Convient pour les énormes bibliothèques.  Ce type de format permet l'utilisation d'une base de données locale ou en ligne.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Ma collection contiendra des </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_Type">
       <item>
        <property name="text">
         <string>Livres</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Films</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="buttonGo">
       <property name="text">
        <string>GO!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
